example.config.yml

```diff
@@ -155,3 +156,6 @@ php_xdebug_default_enable: 0
 solr_version: "4.10.4"
 solr_xms: "64M"
 solr_xmx: "128M"
+
+# Selenium configuration
+server_version: 2.46.0
```

Comment:
Can this be updated (upstream) to a variable with the name of the role prefixed (e.g. `selenium_server_version`)? Similar to Drupal modules, it's best to use a prefixed variable name so there are no conflicts between roles (and to clarify what, exactly, the variable is used for in a large config file).